app/models/webhook_header.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: webhook_headers
+#
+#  id         :integer          not null, primary key
+#  webhook_id :integer
+#  name       :string(255)
+#  value      :string(255)
```

Comment:
This limit of 255 chars might be to low, I'd feel more safe with perhaps 1024.

It's not a strict and founded requirement but a gut feeling.